src/js/component.js

```
@@ -1092,6 +1092,67 @@ class Component {
   }
 
   /**
+   * Get width or height of computed style
+   * @param  {String} widthOrHeight  'width' or 'height'
+   * @return {Number|Boolean} The bolean false if nothing was set
+   * @method currentDimension
+   */
+  currentDimension(widthOrHeight) {
+    let computedWidthOrHeight;
+
+    if (widthOrHeight !== 'width' && widthOrHeight !== 'height') {
+      log.warn('currentDimension only accepts width or height value');
+      return false;
```

Actually, we probably should throw an error in this case.